rust/sbd-server/src/lib.rs

```diff
@@ -5,65 +5,52 @@
 const MAX_MSG_BYTES: i32 = 20_000;
 
 use std::io::{Error, Result};
+use std::net::{IpAddr, Ipv6Addr};
 use std::sync::Arc;
 
 mod config;
 pub use config::*;
 
 mod maybe_tls;
-use maybe_tls::*;
+pub use maybe_tls::*;
 
 mod ip_deny;
 mod ip_rate;
+pub use ip_rate::*;
 
 mod cslot;
+pub use cslot::*;
 
 mod cmd;
 
 /// Websocket backend abstraction.
 pub mod ws {
     /// Payload.
-    pub enum Payload<'a> {
-        /// Immutable slice.
-        Slice(&'a [u8]),
-
-        /// Mutable slice.
-        SliceMut(&'a mut [u8]),
-
```

Comment:
Are these variants removed for simplification?

Reply:
You can't borrow across a BoxFuture return so these values prevented me adding the trait the way I needed to so that we could have multiple websocket implementations plug in. They were barely used and to integrate with tunstenite you have to copy to a Vec anyway